changelogs/fragments/5400-django-manage-deprecations.yml

```
@@ -0,0 +1,2 @@
+deprecated_features:
+  - django_manage - dropping commands ``cleanup``, ``syncdb`` and ``validate`` that have been deprecated in Django long time ago (https://github.com/ansible-collections/community.general/pull/5400).
```

```suggestion
  - django_manage - support for the commands ``cleanup``, ``syncdb`` and ``validate`` that have been deprecated in Django long time ago will be removed in community.general 9.0.0 (https://github.com/ansible-collections/community.general/pull/5400).
```
Also you need a fragment for dropping support for Django < 4.1. (Any reason why you want to drop support for that BTW? Is there some compatibility code for it that can be removed by then?)

---

The main idea here is to keep up with the Django development. Django has just released 4.1, so by the time we reach c.g. 9.0.0, it will be 1.5 years old, and Django will be on version 5.0 and a couple of months short of releasing 5.1. 

The plan is to match roughly the LTS support in Django, to expand the integrations tests and run them against a set of Django versions. Right now I am just setting a starting point for later in the future, so we will have time to work out how this cadence is going to function.

Although I want to document support for a limited number of versions, there is nothing in the code (other than the deprecated commands) that is version-specific.

---

Ok, but then we still need a changelog fragment :)